projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -1,42 +1,47 @@
 
   <PropertyGroup>
-    <VersionPrefix>6.0.0</VersionPrefix>
     <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AssemblyTitle>RabbitMQ Client Library for .NET</AssemblyTitle>
+    <Authors>VMware</Authors>
+    <Company>VMware, Inc. or its affiliates.</Company>
+    <Copyright>Copyright © 2007-2020 VMware, Inc. or its affiliates.</Copyright>
+    <Description>The RabbitMQ .NET client is the official client library for C# (and, implicitly, other .NET languages)</Description>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageTags>RabbitMQ</PackageTags>
     <PackageIcon>icon.png</PackageIcon>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
     <PackageProjectUrl>https://www.rabbitmq.com/dotnet.html</PackageProjectUrl>
-    <PackageLicense>https://www.rabbitmq.com/dotnet.html</PackageLicense>
-    <AssemblyTitle>RabbitMQ Client Library for .NET</AssemblyTitle>
-    <Description>The RabbitMQ .NET client is the official client library for C# (and, implicitly, other .NET languages)</Description>
-    <Company>VMware, Inc. or its affiliates.</Company>
+    <PackageTags>rabbitmq, amqp</PackageTags>
     <Product>RabbitMQ</Product>
-    <Copyright>Copyright © 2007-2020 VMware, Inc. or its affiliates.</Copyright>
-    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <AssemblyOriginatorKeyFile>../rabbit.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
+    <MinVerTagPrefix>v</MinVerTagPrefix>
+    <MinVerVerbosity>detailed</MinVerVerbosity>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageOutputPath>..\..\..\packages</PackageOutputPath>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="..\..\..\gensrc\autogenerated-api-0-9-1.cs" />
-  </ItemGroup>
-
-  <ItemGroup>
     <None Remove="icon.png" />
     <Content Include="icon.png" PackagePath="" />
-  </ItemGroup>
-
-  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>Unit, PublicKey=00240000048000009400000006020000002400005253413100040000010001008d20ec856aeeb8c3153a77faa2d80e6e43b5db93224a20cc7ae384f65f142e89730e2ff0fcc5d578bbe96fa98a7196c77329efdee4579b3814c0789e5a39b51df6edd75b602a33ceabdfcf19a3feb832f31d8254168cd7ba5700dfbca301fbf8db614ba41ba18474de0a5f4c2d51c995bc3636c641c8cbe76f45717bfcb943b5</_Parameter1>
+    </AssemblyAttribute>
+    <None Include="..\..\..\LICENSE">
```

Comment:
This can be simplified to 
```
<None Include="..\..\..\LICENSE" Pack="true" PackagePath="" />
```

I'd also move it up above the `AssemblyAttribute` entry.